### ISSUE TITLE
Disparity Net pretrained weights

### ISSUE THREAD

**Issue by lakshjaisinghani** [ClementPinard/SfmLearner-Pytorch#123]
Hey Clement, 
Thanks for the great code, but I'm wondering what I may be doing wrong.

Link to ipynb (https://github.com/Monash-Connected-Autonomous-Vehicle/UMDE-study/blob/main/visualizer.ipynb)

When I run a single image through the disparity network (using _**dispnet_model_best.pth.tar**_ as the weights). I get something like this (1/output):

![image](https://user-images.githubusercontent.com/45281017/115643494-37020800-a360-11eb-8b94-39f8c7d538fb.png)

The image had been tested with and without normalization before the forward pass, but the result is always the same. I'm wondering what's wrong?

I was also wondering why is the trained pose network requiring 5 sequence images?

**Comment by ClementPinard** [comment]
Hi, it seems to you didn't resize the image and gave the network a full resolution. You need to resize to 416x128 to use the pretrained network.

the pose network has been trained with 5 sequence images, because that is the training that got the best results for disparity estimation. The 3-frames version also works, but not as good.

**Comment by lakshjaisinghani** [comment]
I see, Thanks a ton!
